src/Apps/Legion/Hooks/useRelationships.tsx

```diff
@@ -121,7 +127,9 @@ export const useRelationships = () => {
          * Callback to delete the relationship from state.
          * NOTE: this is not a deep update. It will only delete the root level element
          */
-        deleteRelationship: deleteRelationship
+        deleteRelationship: deleteRelationship,
```

Comment:
instead of a function specific to new, can we do the same thing as entities and just return an object with all the different count types?